convex/webhooks/clerk/handler.ts

```
@@ -0,0 +1,56 @@
+  switch (parsed.type) {
+    case "userCreated":
+      await ctx.runMutation(internal.users.webhooks.userCreated, parsed.args);
+      break;
+    case "userUpdated":
+      await ctx.runMutation(internal.users.webhooks.userUpdated, parsed.args);
+      break;
+    case "userDeleted":
+      await ctx.runMutation(internal.users.webhooks.userDeleted, parsed.args);
+      break;
+    default:
+      const unhandled = parsed as { type: string };
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)